2019-03-19 23:32:25,639, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:25,879, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:25,885, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:29,505, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:29,511, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:29,756, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:29,762, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:33,002, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:33,008, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:33,246, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:33,251, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:36,989, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:36,995, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:37,243, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:37,249, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:40,488, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:40,494, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:40,734, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:40,740, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:43,984, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:43,990, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:44,231, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:44,237, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:49,496, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:49,502, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:49,744, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:49,749, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:52,992, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:52,998, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:53,240, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:53,246, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:56,487, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:56,493, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:56,736, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:56,742, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:32:59,985, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:32:59,991, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:00,251, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:00,257, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:03,504, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:03,507, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:03,740, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:03,743, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:06,979, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:06,984, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:07,219, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:07,224, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:10,473, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:10,478, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:10,716, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:10,721, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:13,964, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:13,968, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:14,202, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:14,206, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:17,443, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:17,448, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:17,692, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:17,697, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:20,945, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:20,951, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:21,191, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:21,197, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:24,438, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:24,444, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:24,683, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:24,689, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:27,929, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:27,935, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:33,145, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:33,152, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:36,395, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:36,399, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:36,637, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:36,641, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:39,886, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:39,891, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:40,126, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:40,132, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:43,374, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:43,380, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:43,621, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:43,626, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:46,865, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:46,871, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:47,111, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:47,117, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:50,363, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:50,369, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:50,611, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:50,617, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:53,857, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:53,861, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:54,097, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:54,103, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:57,344, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:57,348, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:33:57,596, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:33:57,599, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:01,558, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:01,562, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:01,799, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:01,803, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:05,045, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:05,050, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:05,291, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:05,297, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:08,540, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:08,545, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:08,779, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:08,783, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:12,024, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:12,028, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:12,262, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:12,266, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:15,504, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:15,507, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:15,742, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:15,746, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:18,979, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:18,983, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:19,220, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:19,224, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:22,458, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:22,462, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:22,699, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:22,702, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:25,936, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:25,941, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:26,174, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:26,178, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:29,416, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:29,420, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:29,650, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:29,654, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:32,894, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 409 154
2019-03-19 23:34:32,898, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:33,138, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 23
2019-03-19 23:34:33,144, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:36,385, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 23
2019-03-19 23:34:36,390, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:36,629, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 23
2019-03-19 23:34:36,635, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:39,873, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 23
2019-03-19 23:34:39,879, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:40,126, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 23
2019-03-19 23:34:40,131, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:43,203, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:43,210, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:43,445, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:43,450, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:43,686, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:43,692, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:43,932, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:43,938, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:44,183, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:44,189, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:44,425, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:44,431, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:44,671, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:44,677, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:44,916, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:44,919, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:45,157, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:45,161, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:45,396, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:45,400, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:45,631, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:45,634, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:45,869, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:45,873, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:46,104, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:46,108, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:46,341, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:46,345, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:46,577, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:46,580, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:46,813, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:46,817, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:47,050, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:47,055, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:47,290, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:47,298, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:47,533, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:47,539, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:47,778, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:47,783, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:48,025, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:48,031, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:48,272, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:48,278, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:48,515, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:48,520, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:48,765, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:48,771, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:49,012, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:49,017, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:49,255, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:49,262, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:49,503, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:49,509, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:49,744, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:49,750, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:50,007, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:50,011, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:50,248, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:50,251, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:50,488, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:50,493, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:50,739, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:50,743, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:50,978, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:50,982, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:51,214, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:51,219, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:51,457, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:51,464, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:51,954, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:51,958, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:57,179, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:57,185, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:57,426, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:57,432, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:57,669, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:57,673, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:57,912, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:57,918, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:58,157, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:58,163, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:58,398, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:58,403, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:58,638, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:58,644, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:58,883, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:58,888, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:59,124, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:59,130, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:59,368, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:59,375, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:59,615, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:59,622, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:34:59,864, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:34:59,870, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:00,112, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:00,117, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:00,366, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:00,372, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:00,618, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:00,624, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:00,863, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:00,869, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:01,107, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:01,113, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:01,348, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:01,354, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:01,592, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:01,597, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:01,835, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:01,841, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:02,084, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:02,090, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:02,329, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:02,335, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:02,578, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:02,583, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:02,828, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:02,835, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:03,078, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:03,084, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:03,321, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:03,327, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:03,568, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:03,574, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:03,814, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:03,819, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:04,059, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:04,065, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:04,304, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:04,310, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:04,546, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:04,552, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:04,793, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:04,799, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:05,037, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:05,042, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:05,281, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:05,287, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:05,524, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:05,531, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:05,768, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:05,774, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:06,011, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:06,017, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:06,257, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:06,263, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:06,509, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:06,515, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:06,757, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:06,764, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:07,003, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:07,009, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:07,248, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:07,254, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:07,493, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:07,499, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:07,739, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:07,745, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:07,988, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:07,994, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:08,232, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:08,237, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:08,483, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:08,488, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:08,726, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:08,733, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:08,973, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:08,978, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:09,218, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:09,224, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:09,464, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:09,471, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:09,715, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:09,721, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:09,961, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:09,968, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:10,207, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:10,213, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:10,452, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:10,458, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:10,695, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:10,699, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:10,935, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:10,939, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:11,178, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:11,182, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:11,417, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:11,421, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:11,654, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:11,658, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:11,930, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:11,935, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:12,168, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:12,174, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:12,414, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:12,418, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:12,656, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:12,659, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:12,898, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:12,902, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:13,140, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:13,144, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:13,384, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:13,390, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:13,628, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:13,632, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:13,881, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:13,887, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:14,127, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:14,133, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:14,373, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:14,380, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:14,620, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:14,626, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:14,866, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:14,872, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:15,113, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:15,119, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:15,365, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:15,371, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:15,610, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:15,616, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:15,858, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:15,865, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:16,105, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:16,111, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:16,353, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:16,359, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:16,604, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:16,607, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:16,842, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:16,848, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:17,089, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:17,094, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:17,336, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:17,343, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:17,581, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:17,585, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:17,819, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:17,822, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:18,056, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:18,060, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:18,295, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:18,300, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:18,543, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:18,549, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:18,788, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:18,794, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:19,035, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:19,041, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:19,295, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:19,301, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:19,542, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:19,547, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:19,790, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:19,796, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:20,045, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:20,051, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:20,292, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:20,297, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:20,533, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:20,539, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:20,777, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:20,783, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:21,027, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:21,033, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:21,270, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:21,276, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:21,517, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:21,523, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:21,765, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:21,771, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:22,012, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:22,018, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:27,239, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:27,245, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:27,481, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:27,487, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:27,731, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:27,737, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:28,581, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:28,587, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:28,832, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:28,838, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:29,077, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:29,082, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:29,332, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:29,339, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:29,583, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:29,589, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:29,833, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:29,839, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:30,079, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:30,083, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:30,316, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:30,322, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:30,564, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:30,570, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:30,808, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:30,814, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:31,456, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:31,462, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:31,702, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:31,708, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:31,946, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:31,953, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:32,192, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:32,198, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:32,435, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:32,442, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:32,682, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:32,688, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:32,928, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:32,934, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:33,176, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:33,182, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:33,439, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:33,445, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:33,683, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:33,689, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:33,928, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:33,934, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:34,179, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:34,184, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:34,430, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:34,435, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:34,676, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:34,682, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:34,919, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:34,925, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:35,171, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:35,177, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:35,419, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:35,424, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
2019-03-19 23:35:35,991, DEBUG, urllib3.connectionpool, connectionpool.py, _make_request, сообщение: https://api.telegram.org:443 "GET /bot408697626:AAHW5D48WVfee5zTT3nZmQ58OPsauhLhVzA/getUpdates HTTP/1.1" 200 350
2019-03-19 23:35:35,997, DEBUG, urllib3.connectionpool, connectionpool.py, _new_conn, сообщение: Starting new HTTPS connection (1): api.telegram.org:443
